model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -84,6 +85,10 @@ class LicenseInfoResolver(
             license.builder().apply {
                 sources += LicenseSource.DECLARED
 
+                licenseInfo.declaredLicenseInfo.processed.spdxExpression?.let {
+                    originalExpressions.add(it)
```

Comment:
Can we use `+=` instead of `.add()` here?

Reply:
No, that doesn't work:
`Assignment operators ambiguity. All these functions match`
`plus` from `Set` and plusAssign from `MutableCollection` would fit. That why `.add` was used explicitly.